test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/013_sequence_switches_of_accessMode.lua

```diff
@@ -53,13 +54,17 @@ runner.Title("Default -> ASK_DRIVER")
 runner.Step("Enable RC from HMI with ASK_DRIVER access mode", commonRC.defineRAMode, { true, "ASK_DRIVER"})
 for _, mod in pairs(commonRC.modules)  do
   runner.Step("Activate App2", commonRC.activateApp, { 2 })
-  runner.Step("Check module " .. mod .." App2 SetInteriorVehicleData allowed", commonRC.rpcAllowed, { mod, 2, "SetInteriorVehicleData" })
+  runner.Step("Check module " .. mod .." App2 SetInteriorVehicleData allowed",
+      commonRC.rpcAllowed, { mod, 2, "SetInteriorVehicleData" })
   runner.Step("Activate App1", commonRC.activateApp)
-  runner.Step("Check module " .. mod .." App1 SetInteriorVehicleData allowed with driver consent", commonRC.rpcAllowedWithConsent, { mod, 1, "SetInteriorVehicleData" })
+  runner.Step("Check module " .. mod .." App1 SetInteriorVehicleData allowed with driver consent",
+      commonRC.rpcAllowedWithConsent, { mod, 1, "SetInteriorVehicleData" })
   runner.Step("Activate App2", commonRC.activateApp, { 2 })
-  runner.Step("Check module " .. mod .." App1 ButtonPress allowed with driver consent", commonRC.rpcAllowedWithConsent, { mod, 2, "ButtonPress" })
+  runner.Step("Check module " .. mod .." App1 ButtonPress allowed with driver consent",
+      commonRC.rpcAllowedWithConsent, { mod, 2, "ButtonPress" })
   runner.Step("Activate App1", commonRC.activateApp)
-  runner.Step("Check module " .. mod .." App1 ButtonPress allowed with driver consent", commonRC.rpcAllowedWithConsent, { mod, 1, "ButtonPress" })
+  runner.Step("Check module " .. mod .." App1 ButtonPress allowed with driver consent",
+      commonRC.rpcAllowed, { mod, 1, "ButtonPress" })
```

Comment:
@iCollin Function `rpcAllowedWithConsent` was replaced with `rpcAllowed` in this step because Multiple module feature introduced new rules for ASK_DRIVER remote access mode with caching of driver decisions (see https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0221-multiple-modules.md#how-to-use-the-new-parameter-for-permission-control section started with "The driver's yes/no decision shall be cached")
So at the time of mentioned step driver decision will be cached and allocation of module processed without asking driver for consent.